### DISCUSSION TITLE
Scheduled reminders do not show all available repos in "Only select repositories" dropdown

### DISCUSSION THREAD

**Discussion by jennifer-shehane** [community/community#23247] [New to GitHub]
<p>When setting up a scheduled reminder for a Team within GitHub, I only want to choose selected repositories to be reminded of (our team has access to 100+ repos - we don’t want notifications for these).</p>
<p>But when I choose the ‘Only select repositories’, it lists only 3 repos and it does  <strong>not</strong> list the repos that I want to set reminders for at all. </p>
<p>How can I select only a few repos in the scheduled Slack reminders? I’d like to move off of our 3rd party reminder bot, but this is blocking. </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jennifer-shehane">@jennifer-shehane</a>,</p>
<p>Thank you for being here! It might be useful to check that the Slack App installation includes all repositories you wish to configure to send scheduled reminders for pull requests. The repositories shown in the drop down list are the result of performing a union of all repositories where the team has <code>admin</code> permissions, and those which have the Slack App installed.</p>

**Reply by YuanliangYu** [reply]
Hi @AndreaGriffiths11 , thanks for your answer. Would you mind to elaborate how to check if a repo has the Slack App installed and how to  install the Slack App on a specific repo? Thanks again!

**Comment by brandonb927** [comment]
<p>THANK YOU! This would be incredibly helpful if it was added to these two pages because I was tearing my hair out trying to figure out why only 1 repository was listed as available to send messages to :angry: </p>
<p>- <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-organization" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-organization</a><br>
- <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-scheduled-reminders-for-your-team</a></p>

**Comment by jennifer-shehane** [comment]
<p>Thank you! This solved it completely for our team.</p>

**Comment by niltonvasques** [comment]
I also lost some hours trying to figure out the reason for some repos not been listed. I only found this thread when I started to open a ticket on github support. 

Would be very helpful to have it highlighted in the Scheduled Reminders Wiki:  https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team

**Comment by randradejc** [comment]
Having issues with this, not being able to list any repos at all:
- I'm both member of `admin` team on a given repo and the `maintainer` of said team
- I have the Slack integration working at an organization level, as well as for myself (I can get scheduled reminders)

No repos are listed or if I pick all repos, saving the settings gives me:
> There was an error: No repositories visible with the selected settings and teams

I do not have visibility on the organization settings, but I didn't see anything in the documentation other than the Slack integration itself at the org-level.

Thoughts on what step/setting is missing to make this work?

**Reply by blisteringherb** [reply]
Did you happen to find a solution? I'm having a similar issue.

Edit:
To fix this I navigated to the Github Apps for a repository that Slack already had access to and clicked "Configure". On that page, you can add additional repositories that you want Slack to have access to.

**Reply by YuanliangYu** [reply]
@blisteringherb Hi Morgan, would you mind to elaborate how to navigate to the Configure page? More specifically, how could I know if I have a repo that Slack already has had access to? If I don't have one, how should I add the very first? Thanks!

**Reply by gcbeltramini** [reply]
All steps that I took:

1. Enable the GitHub + Slack integration: https://slack.github.com/ --> `Add to Slack` --> `Allow`
1. Connect to GitHub by running `/github signin` on Slack (e.g., in a DM to yourself or to GitHub app).
1. A GitHub repo admin needs to configure the Slack plugin on that repo ([reference](https://github.com/orgs/community/discussions/23247#discussioncomment-5295753)) following the steps below. Otherwise, the repository will require the approval from an admin.
1. To select which repos Slack will have access to, run `/github subscribe my-org/my-repo` on Slack (where `my-org/my-repo` can be literally anything). You will receive the following message only visible to you:

    ```text
    Either the app isn't installed on your repository or the repository does not exist. Install it to proceed.
    Note: You need to be an organization owner to install the app (or ask one to install it for you).
    ```

    Click on the button `Install GitHub App` --> in the page that opens in the browser, click on `Configure` next to your organization name --> select the repositories

    Alternatively, if you already know a repo that has Slack integration, go to `https://github.com/my-org/my-repo/settings/installations` (you probably need to have admin permission in the repo) --> click on `Configure` next to the Slack icon.

**Reply by ojintoad** [reply]
I had this issue for a different root cause. Here's a claude generated summary.

## The Problem

When setting up GitHub scheduled reminders for a team, you get the error:

> "There was an error: No repositories visible with the selected settings and teams"

Even though the repository dropdown shows repos your team can access, saving the reminder configuration fails.

## Root Cause

GitHub's scheduled reminders feature only recognizes **direct, explicit team-level repository access**. It does not count repositories that teams can access through organizational roles or permissions.

Your team may have full access to repositories through their org role, but the scheduled reminders validator treats that as invisible access.

## Solution

Grant your team **direct access** to at least one repository:

1. Go to the repository's **Settings → Access → Teams and people**
2. Add your team with the desired permission level
3. Attempt to create the scheduled reminder again

Once your team has direct access to at least one repository, the scheduled reminders feature will recognize the team's access and allow you to configure reminders. You can clean up the direct access once the reminder is created.

## Why This Happens

This appears to be a GitHub limitation where the scheduled reminders feature uses a narrower permission model than the rest of GitHub's infrastructure. The validator doesn't account for organizational role-based repository access, only direct team assignments.

**Comment by sglajchEG** [comment]
EDIT: OK You need an admin to configure the slack plugin itself, but the team does NOT need to have admin access to the repo.
I think the wording in the suggested solution was off.  I should have known better :sweat_smile: 
~Why in the world do you need _an entire team to have admin permissions to the repo_ just in order to put a reminder about open PRs for that team?
That needs to change.~
